hazelcast/include/hazelcast/client/serialization/pimpl/compact.h

```diff
@@ -1522,6 +1529,29 @@ struct HAZELCAST_API field_operations
     static field_kind_based_operations get(enum field_kind field_kind);
 };
 
+namespace rabin_finger_print
+{
+
+/**
+ * We use uint64_t for computation to match the behaviour of >>> operator
+ * on java. We use >> instead.
+ */
+constexpr uint64_t INIT = 0xc15d213aa4d7a795L;
+
+uint64_t HAZELCAST_API
+fingerprint64(uint64_t fp, byte b);
+
+uint64_t HAZELCAST_API
+fingerprint64(uint64_t fp, int v);
```

Comment:
```suggestion
fingerprint64(uint64_t fp, int32_t v);
```
Java `int` is `int32_t`.

Reply:
Fixed.